test/unit/lib/api-client.js

```diff
@@ -0,0 +1,325 @@
+            endpoint: '/api/checkout',
+            response
+        }).then(() => {
+            return checkOutItems('123456', ['iGwEZUvfA', 'iGwEZVHHE'], 'abcdef');
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                studentId: '123456',
+                itemAddresses: ['iGwEZUvfA', 'iGwEZVHHE'],
+                adminCode: 'abcdef'
+            });
+        });
+    });
+
+    it('createItem', () => {
+        let response = {
+            status: 'success',
+            data: {
+                address: 'iGwEZUvfA',
+                modelName: 'Resistor'
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'post',
+            endpoint: '/api/item',
+            response
+        }).then(() => {
+            return createItem('m8y7nEtAe');
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                modelAddress: 'm8y7nEtAe'
+            });
+        });
+    });
+
+    it('createModel', () => {
+        let response = {
+            status: 'success',
+            data: {
+                address: 'm8y7nEtAe',
+                name: 'Resistor',
+                description: 'V = IR',
+                manufacturer: 'Live',
+                vendor: 'Mouzer',
+                location: 'Shelf 14',
+                isFaulty: false,
+                faultDescription: '',
+                price: 10.50,
+                count: 20
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'post',
+            endpoint: '/api/model',
+            response
+        }).then(() => {
+            return createModel('Resistor', 'V = IR', 'Live', 'Mouzer', 'Shelf 14', false, '', 10.50, 20);
+        }).then(data => {
+            assert.deepEqual(data, response.data);
+            MockServer.validate({
+                name: 'Resistor',
+                description: 'V = IR',
+                manufacturer: 'Live',
+                vendor: 'Mouzer',
+                location: 'Shelf 14',
+                isFaulty: false,
+                faultDescription: '',
+                price: 10.50,
+                count: 20
+            });
+        });
+    });
+
+    it('deleteItem', () => {
+        let response = {
+            status: 'success',
+            data: {
+                items: [{
+                    address: 'iGwEZVHHE',
+                    modelAddress: 'm8y7nEtAe',
+                    status: 'AVAILABLE'
+                }],
+                modelName: 'Resistor'
+            }
+        };
+        return MockServer.listen({
+            port: 8080,
+            method: 'delete',
+            endpoint: '/api/item',
+            response
+        }).then(() => {
+            return deleteItem({
+                address: 'iGwEZUvfA',
+                modelAddress: 'm8y7nEtAe'
```

Comment:
`modelAddress` is not required, we only need `itemAddress`.

This goes back to https://github.com/TheFourFifths/consus-client/blob/api-testing-TFF-117/src/lib/api-client.js#L95. Server-side, we also don't need to send `modelAddress` because the item is looked up only based on the _item_ address (https://github.com/TheFourFifths/consus/blob/dev/src/api/item.js#L41).

Nit: I don't think `modelAddress` should be included in the "happy path" test since it is not _required_.